bot/server.go

```diff
@@ -58,25 +58,27 @@ func (b *Bot) endpointHandler(w http.ResponseWriter, r *http.Request, chnl chann
 }
 
 func (b *Bot) slackMessageEvents() {
-	receiveChan := make(chan messages.Receive)
-
-	go b.Channels.Slack.ReceiveMessages(receiveChan)
-
-	go func() {
-		for receiveMsg := range receiveChan {
-			answers, err := b.Answer(receiveMsg.Question)
-			if err != nil {
-				log.Error(err)
-				return
+	if b.Channels.Slack != nil {
```

Comment:
I added this condition to skip this process if there is no Slack channel configuration